### DISCUSSION TITLE
I want all buttons in addonbar_vertical.uc.js to be in classic mode.

### DISCUSSION THREAD

**Discussion by anzuno** [Aris-t2/CustomJSforFx#121] [Q&A]
Hello.
Added a CSS file for classic buttons to addonbar_vertical.uc.js using "buttons_on_navbar_classic_appearance.css". Uses userChrome.css.
The add-on button will not change. Can we make all the buttons classic?

This is the part that was changed.
#nav-bar → #addonbar_v
#nav-bar #nav-bar → #addonbar_v #addonbar_v

Added for button spacing.
#addonbar_v > toolbarbutton,
#addonbar_v > toolbar .toolbarbutton-1 {
 margin-top: 4px !important; 
 margin-right: 0px !important; 
}
![buttons_on_navbar_classic_appearance](https://github.com/user-attachments/assets/82beee9d-7184-4100-906f-f782977b181c)

**Comment by Aris-t2** [comment]
[buttons_on_navbar_classic_appearance.css](https://github.com/Aris-t2/CustomCSSforFx/blob/master/current/css/buttons/buttons_on_navbar_classic_appearance.css)

Get the base file and replace rules like this:

1st step
`#nav-bar #nav-bar-customization-target` >>> `#addonbar_v`

2nd step
`#nav-bar ` >>> `#addonbar_v`

**Reply by anzuno** [reply]
Thank you for your quick reply despite your busy schedule. It worked well. I appreciate it.

Added to adjust the button spacing of add-ons.
#addonbar_v .unified-extensions-item-action-button {
 margin-top: 2px !important; 
 margin-bottom: -2px !important;
 margin-right: 0px !important; 
}
![aaa](https://github.com/user-attachments/assets/1f09f8bf-5d91-46b3-b661-63c1a4897c9a)
